CRM/Import/ImportProcessor.php

```diff
@@ -415,8 +436,8 @@ public function getImporterObject() {
       $this->getFieldWebsiteTypes()
       // $mapperRelatedContactWebsiteType = []
     );
+    $importer->setUserJobID($this->getUserJobID());
     $importer->init();
-    $importer->_contactType = $this->getContactType();
```

Comment:
Note that per class code comments - this function is only called from unit tests